pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -35,7 +35,7 @@
 
 # Note: the hardening flags are part of the bintools-wrapper, rather than
 # the cc-wrapper, because a few of them are handled by the linker.
-, defaultHardeningFlags ? with stdenvNoCC; [
+, defaultHardeningFlags ? [
```

Comment:
Here the `with` could also be moved into the `optional` condition.

Reply:
Moved; that's better.